### DISCUSSION TITLE
can you set a max number of executed chat limit per user or department?

### DISCUSSION THREAD

**Discussion by comsyspro** [LiveHelperChat/livehelperchat#1871] [General]
is it possible that you can define a max chat number per user and/or department, so you can say that a user/department can only do x number of chats in x number of days/weeks/months? if this is not implemented could you please add these settings?

**Comment by remdex** [comment]
Yes this can be developed, if you wish to sponsor it - you can contact me via discord https://discord.gg/YsZXQVh

I'm just to bussy with clients' orders to do free things these days unless they are bugs or high value for community items...
